src/styles.css

```diff
@@ -68,3 +71,13 @@ body {
     background-color: rgba(255, 255, 255, 0.4);
   }
 }
+
+
+@keyframes scale-up-center {
+  0% {
+    transform: scale(0.8);
```

Comment:
decrease the size variation, 0.8 is too small and 1.2 is too large, maybe 0.9 to 1.0 would be better.

Reply:
ok